tss-esapi/src/context/tpm_commands/attestation_commands.rs

```diff
@@ -6,13 +6,13 @@ use crate::{
         Attest, AttestBuffer, CreationTicket, Data, Digest, PcrSelectionList, Signature,
         SignatureScheme,
     },
-    tss2_esys::{Esys_Certify, Esys_Quote},
+    tss2_esys::{Esys_Certify, Esys_GetTime, Esys_Quote},
     Context, Result, ReturnCode,
 };
 use log::error;
 use std::convert::TryFrom;
 use std::ptr::null_mut;
-use tss_esapi_sys::Esys_CertifyCreation;
+use tss_esapi_sys::{Esys_CertifyCreation, ESYS_TR_RH_ENDORSEMENT};
```

Comment:
I know it is not you who added this but. I do not think we should import using the ```tss_esapi_sys``` but instead use the ```tss_esys``` alias. So the ```Esys_CertifyCreation``` should be moved.